### DISCUSSION TITLE
Mac OS Monterey

### DISCUSSION THREAD

**Discussion by tdan71** [soundshed/soundshed-app#52] [General]
Hi, just downloaded soundshed and it wont connect to my M1 mac mini running Montery. No issues with a macbook running High Sierra.

Thanks

**Comment by webprofusion-chrisc** [comment]
Thanks, can you try https://app.soundshed.com/ instead using Google Chrome or Microsoft Edge (it may also work in Safari but it requires Web Bluetooth which may not be available).

**Reply by arnoldpistorius** [reply]
I have the same problem as tdan71. The webapp in chrome works. But I prefer the application.

**Reply by webprofusion-chrisc** [reply]
Thanks, If someone wanted to port https://github.com/tinyprinter/node-bluetooth-serial-port to M1 when that might work. Our minor fork is here: https://github.com/webprofusion-chrisc/node-bluetooth-serial-port

**Reply by webprofusion-chrisc** [reply]
Alternatively if we can move the electron version to use web BLE (like the web version does) that might be simpler, then everything can use the same bluetooth path.
